R/logger.R

```diff
@@ -272,13 +272,18 @@ log_level <- function(level, ..., namespace = NA_character_,
     }
 
     definitions <- get_logger_definitions(namespace, .topenv = .topenv)
+    level <- validate_log_level(level)
 
     for (definition in definitions) {
 
+        if (level > definition$threshold) {
```

Comment:
Actually, I think we want to skip logging when the currently used log level is *BELOW* the threshold, no?

```suggestion
        if (level < definition$threshold) {
```

Reply:
You obviously know the code base better, but looking at https://github.com/daroczig/logger/blob/master/R/levels.R it seems that the higher the level the lower the "seriousness", so for example if level is `DEBUG` (`500`) and threshold is on `INFO` (`400`), you want to skip -> so skipping when `level > threshold` makes sense to me. 

This is also the current implementation if you look at the `logger()` function (line `36` in https://github.com/daroczig/logger/blob/master/R/logger.R).

Reply:
jozefhajnalaatradius 2 minutes ago  • 
You obviously know the code base better, but looking at https://github.com/daroczig/logger/blob/master/R/levels.R it seems that the higher the level the lower the "seriousness", so for example if level is DEBUG (`500`) and threshold is on INFO (`400`), you want to skip -> so skipping when `level > threshold` makes sense to me.

This is also the current implementation of "skipping" if you look at the `logger()` function : https://github.com/daroczig/logger/blob/f9bff45bf00e46c5bd98ce405bcf3d2306f85776/R/logger.R#L36

Reply:
Oh, right, sorry -- no more late night reviews :man_facepalming: